hmy/staking.go

```diff
@@ -330,6 +330,8 @@ func (hmy *Harmony) GetValidatorInformation(
 	if defaultReply.CurrentlyInCommittee {
 		defaultReply.Performance = &staking.CurrentEpochPerformance{
 			CurrentSigningPercentage: *computed,
+			Epoch:                    hmy.BeaconChain.CurrentBlock().Header().Number().Uint64(),
+			Block:                    hmy.BeaconChain.CurrentBlock().Header().Epoch().Uint64(),
```

Comment:
There is a small chance that block could change from previous call and getting twice from mutex doesn't make it faster.